src/query-builder/QueryFormatter.ts

```diff
@@ -0,0 +1,94 @@
+import {ObjectLiteral} from "../common/ObjectLiteral";
+import {Connection} from "../connection/Connection";
+import {EntityTarget} from "../common/EntityTarget";
+import {AliasesLiteral, QueryFormatBuilder} from "./QueryFormatBuilder";
+
+/**
+ * Allows to build full sql queries 
+ */
+export class QueryFormatter {
```

Comment:
What is the purpose of this class if it seems to just wrap everything that the `QueryFormatBuilder` does?

Reply:
yes, yes it looks a little weird. I'll explain now. I tried not to change the code of QueryBuilder . so I created "CustomQueryBuilder" (that extends "QueryBuilder")  and rewrote some of the methods. but CustomQueryBuilder has a lot of unnecessary methods for this issue, so I made  class "CustomQuery" that takes some of the methods needed.  Maybe you can suggest a better solution